static/src/javascripts/projects/common/modules/commercial/contributions-liveblog-utilities.js

```diff
@@ -1,160 +1,29 @@
-import { logView } from 'common/modules/commercial/acquisitions-view-log';
 import $ from 'lib/$';
-import mediator from 'lib/mediator';
-import { elementInView } from 'lib/element-inview';
-import fastdom from 'lib/fastdom-promise';
-import {submitInsertEvent, submitViewEvent} from "common/modules/commercial/acquisitions-ophan";
 import { mountDynamic } from "@guardian/automat-modules";
 
-let isAutoUpdateHandlerBound = false;
-const INSERT_EPIC_AFTER_CLASS = 'js-insert-epic-after';
```

Comment:
No longer support manually placing an epic in a specific spot, as this isn't being used.
So now we only place a single epic in a randomly chosen spot